plugin/src/test/groovy/net/dzikoysk/funnyguilds/user/TestUserProfile.java

```diff
@@ -0,0 +1,42 @@
+package net.dzikoysk.funnyguilds.user;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TestUserProfile extends MissingUserProfile implements UserProfile {
```

Comment:
Niezbyt to robi sens, żeby TestProfile dziedziczył po MissingProfile. Niech UserProfile ma default metody, i tyle.